journal/journal/tests/test_views.py

```diff
@@ -0,0 +1,59 @@
+from journal.views import my_view, compose, entry_detail, edit_entry
+from pyramid.testing import DummyRequest
+
+
+def test_my_view(dbtransaction, new_entry):
+    """Test my_view retunrs a lists entries in the response object."""
+    req = DummyRequest()
+    resp = my_view(req)
+    entries_dict = resp['entries']
+    assert entries_dict[0] == new_entry
```

Comment:
This is good.  Can you write a test with three entries that have different `created` datetimes, that asserts that they are in fact listed in reverse chronological order.